hieradata/common.yaml

```diff
@@ -562,6 +558,9 @@ govuk::apps::local_links_manager::unicorn_worker_processes: "4"
 
 govuk::apps::mapit::enabled: true
 
+govuk::apps::rummager::enable_rummager: false
+govuk::apps::search_api::rummager_enable: false
```

Comment:
I'm finding these changes a little hard to follow... Will setting this here have any effect, given the change to the `search` node class in this same file?

Reply:
(this applying to both lines I guess, particularly the first one)

Reply:
I think I should probably leave the `node_class` how it is for now: https://github.com/alphagov/govuk-puppet/blob/3e0492ca60ff066db99a02140f38e3729a211533/modules/govuk/manifests/node/s_base.pp#L46-L53
I'll update the PR